src/open-data/Services.csv

```diff
@@ -1,314 +1,314 @@
-microsoft.web,microsoft.web/serverfarms,Azure App Service,Web,Microsoft,Cloud Provider
-Microsoft.EventGrid,microsoft.web/sites,Azure App Service,Web,Microsoft,Cloud Provider
-Microsoft.Web,microsoft.web/sites,Azure App Service,Web,Microsoft,Cloud Provider
-microsoft.web,microsoft.web/staticsites,Azure App Service,Web,Microsoft,Cloud Provider
-Microsoft.WindowsIoT,microsoft.windowsiot/deviceservices,Windows for IoT,Internet of Things,Microsoft,Cloud Provider
-NGINX.NGINXPLUS,nginx.nginxplus/nginxdeployments,NGINX Plus,Networking,NGINX,Marketplace
-PlayFabBillingService,playfabbillingservice/blobs,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/catalogrequests,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/compute,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/egress,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/events,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/executiontimegbs,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/filestorage,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/flatfee,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/insightscredits,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/inventoryreadrequests,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/inventorywriterequests,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/itemstorage,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/lobbyrequests,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/matchmakingrequests,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/messages,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/partyconnectivity,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/playstream,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/reads,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/storage,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/telemetry,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/totalexecution,Azure PlayFab,Media,Microsoft,Cloud Provider
-PlayFabBillingService,playfabbillingservice/writes,Azure PlayFab,Media,Microsoft,Cloud Provider
+ConsumedService;ResourceType;ServiceName;ServiceCategory;PublisherName;PublisherType;Location;Service Model

```

Comment:
Can you revert the switch to semicolons? We should find a different word than "Location". Mind chatting with Copilot to see if she has any suggestions? 😅 I'd do it but I'm in the middle of chatting with her about a deck I'm trying to write so trying not to get too distracted 🙂

Reply:
Replaced it by "environment"